src/BigQuery/BigQueryClient.php

```diff
@@ -20,6 +20,7 @@
 use Google\Cloud\BigQuery\Connection\ConnectionInterface;
 use Google\Cloud\BigQuery\Connection\Rest;
 use Google\Cloud\ClientTrait;
+use Psr\Cache\CacheItemPoolInterface;
```

Comment:
Is this use statement required for phpdoc to parse line 67? Unfortunate that we have to import the namespace just for a comment block.